website/docs/tutorials/managing-access.md

```diff
@@ -81,7 +95,7 @@ import {
 const litDatasetWithAcl = await unstable_fetchLitDatasetWithAcl(
   "https://example.com"
 );
-const accessByAgent = unstable_getAgentAccessModesAll(litDatasetWithAcl);
+const accessByAgent = unstable_getAgentAccessModesAll(litDatasetWithAcl.acl);
```

Comment:
So if we do this, we lose the ability to eventually make the ACL data part of the Resource itself. Not sure if that's desirable... (As in, I think Pat might have something to say about that :P )